pkg/apiserver/authentication/options/authenticate_options.go

```diff
@@ -68,7 +72,10 @@ func NewAuthenticateOptions() *AuthenticationOptions {
 func (options *AuthenticationOptions) Validate() []error {
 	var errs []error
 	if len(options.JwtSecret) == 0 {
-		errs = append(errs, fmt.Errorf("jwt secret is empty"))
+		errs = append(errs, errors.New("JWT secret should not be empty"))
```

Comment:
```suggestion
		errs = append(errs, errors.New("JWT secret MUST not be empty"))
```
When talking about something that is forbidden, use `MUST not`, use `MUST` when being postitive.

Reply:
fixed